### DISCUSSION TITLE
What's this rye on PyPI?

### DISCUSSION THREAD

**Discussion by bittner** [astral-sh/rye#59] [Q&A] [answered]
I was a little irritated when I found `rye` [on PyPI](https://pypi.org/project/rye/) just now. (Yes, _this_ rye is written in Rust, I understand.)

Interestingly, Paul Becotte's tool does something (roughly) similar.

Most likely, there is no connection between those projects. @mitsuhiko, can you confirm?

Would be interesting to know why both projects are called "rye", anyway.

----

P.S.: Looks funny when combined with Rust-rye. What will this code do? :smile: 

```
cargo install rye && rye add rye && rye sync && rye
```

**Comment by mitsuhiko** [comment, marked as answer]
I picked `rye` because it somewhat rhumes with "py", is short enough to type. Since this was hanging around on my computer for a while, I never considered if the name is used on a package index. There is no connection between them.
